### PR TITLE
Draft: Fairfood

### LINKED ISSUE #2: Fairfood
- [ ] Login to Fairfood page
- [ ] Fetch rss
- [ ] Parse rss to foodsoft_articles
- [ ] ...

### PR DESCRIPTION
**Pull request by Nalaxon**
Closes #2 

Feedback welcome @twothreenine 

# Todo
- [x] logon to fairfood.org
- [x] fetch and parse rss
- [ ] convert to `foodsoft_article`
- [ ] fetch origin, manufacturer
- [ ] price_net, ... conversion
- [ ] Test & Accept @twothreenine

### CODE REVIEW

File: script_krautkoopf_Fairfood_import.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import os
```
Comment (twothreenine) [inline script_krautkoopf_Fairfood_import.py:9]: don't we have to import selenium?

**Comment by twothreenine** [comment]
Great!
I couldn't find the time to test this for now, but go on if you want to. Or let me merge it if you want me to continue.
We'll need it mid-March so it's not very urgent, should be doable now.
Please also add requirements in the Readme.

**Comment by Nalaxon** [comment]
> Great! I couldn't find the time to test this for now, but go on if you want to. Or let me merge it if you want me to continue. We'll need it mid-March so it's not very urgent, should be doable now. Please also add requirements in the Readme.

I updated README and imports.
I think it is best if you continue, it is not clear for me, how you added the articles in the first place.
If you like, we could split the work (e.g.: I work on a method to parse the origin, and so forth...)

I think, I don't have write access to your repo, thus I can't merge :) (which is no problem for me)

**Comment by twothreenine** [comment]
> it is not clear for me, how you added the articles in the first place

Odd but good to know, I thought the [simple example](https://github.com/twothreenine/krautkoopf_lieferscraping/blob/7467241ae331ec6d1f012c5bd1728d41951bcc4d/script_generic_test_import.py#L36:L47) was very clear, or did you overlook that?

> I think, I don't have write access to your repo, thus I can't merge :) (which is no problem for me)

Since it was in "draft" status I couldn't merge it and I thought you had to change the status first, but I found out how to do that now :)

**Pull request merged**